### PR TITLE
Mysql plugin integration

### PR DESCRIPTION
**Pull request by geek-nupur**
Initial commit for mysql plugin integration

### CODE REVIEW

File: app/server/appsmith-plugins/mysqlPlugin/src/main/java/com/external/plugins/MySqlPlugin.java [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+            }
+            try {
+                Connection connection = DriverManager.getConnection(url, properties);
+                connection.setReadOnly(
+                        configurationConnection != null && READ_ONLY.equals(configurationConnection.getMode()));
+                return Mono.just(connection);
+
+            } catch (SQLException e) {
+                return pluginErrorMono("Error connecting to MySql.", e);
+
```
Comment (trishaanand) [inline app/server/appsmith-plugins/mysqlPlugin/src/main/java/com/external/plugins/MySqlPlugin.java:154]: Please format the code using the IDE's formatting feature.

File: app/server/appsmith-plugins/postgresPlugin/plugin.properties [hunk 2]
```diff
@@ -1,5 +0,0 @@
-plugin.id=postgres-plugin
```
Comment (mohanarpit) [inline app/server/appsmith-plugins/postgresPlugin/plugin.properties:1]: Why did we delete this file? Is it no longer required?

File: app/server/appsmith-plugins/mysqlPlugin/src/test/java/com/external/plugins/MySqlPluginTest.java [hunk 3]
```diff
@@ -0,0 +1,112 @@
+package com.external.plugins;
```
Comment (mohanarpit) [inline app/server/appsmith-plugins/mysqlPlugin/src/test/java/com/external/plugins/MySqlPluginTest.java:1]: Can we add more test cases for other functions like `validateDatasource`, `testDatasource` etc ?

File: app/server/appsmith-plugins/mysqlPlugin/src/main/java/com/external/plugins/MySqlPlugin.java (hunk 1 above)
Comment (geek-nupur) [reply app/server/appsmith-plugins/mysqlPlugin/src/main/java/com/external/plugins/MySqlPlugin.java:154]: Hi Trisha! The code is already formatted using the default formatter available in my IDE. Do you need formatting with some special formatter then please share the same. Thanks

File: app/server/appsmith-plugins/postgresPlugin/plugin.properties (hunk 2 above)
Comment (geek-nupur) [reply app/server/appsmith-plugins/postgresPlugin/plugin.properties:1]: I added this file by reading the other plugins format. But as of now I didn't feel it is required. Didn't feel it was required to successfully compile /build the code and run the test cases.

File: app/server/appsmith-plugins/mysqlPlugin/src/test/java/com/external/plugins/MySqlPluginTest.java (hunk 3 above)
Comment (geek-nupur) [reply app/server/appsmith-plugins/mysqlPlugin/src/test/java/com/external/plugins/MySqlPluginTest.java:1]: I have added some test cases for validateDatasource. Will check for other functions too what test cases can be written. Thanks

File: app/server/appsmith-plugins/postgresPlugin/plugin.properties (hunk 2 above)
Comment (mohanarpit) [reply app/server/appsmith-plugins/postgresPlugin/plugin.properties:1]: But you've deleted this from the postgresPlugin. Why change code in some other plugin? This is required in order to run the plugin via IntelliJ

Comment (geek-nupur) [reply app/server/appsmith-plugins/postgresPlugin/plugin.properties:1]: Oh Sorry! I had to delete the file in mysql plugin. I have added it again

**Pull request merged**